steps/waldo-upload/2.0.0/step.yml

```diff
@@ -0,0 +1,99 @@
+title: Waldo Upload
```

Comment:
Please update the title to be similar to *Upload to Waldo* in the next release (https://devcenter.bitrise.io/contributors/create-your-own-step/#the-title).